### ISSUE TITLE
Work out how to deal with extra useful links

### ISSUE THREAD

**Issue by robintw** [DeepBlueCLtd/LegacyMan#422]
In `unit_anchors.html` we have a top-level div that looks like this:

```
<div style="position: absolute; visibility: visible; left: 865px; top: 16px; width: 127px">
<a href="Content/Images/dummy reference.pdf" target="_blank">Unit Anchors Study</a>
</div>
```

This link seems useful, and we want to include it, but at the moment it is being excluded as it's not a link inside a PageLayer div.

We should check how this appears in the real data, and work out how we deal with it.

* [ ] Inspect instance that occurs in A3
* [ ] Look for instances in other files, check for pattern
* [ ] Establish logic that prevents non useful links being parsed (back, corporate logo, etc)

**Comment by IanMayo** [comment]
There is only one instance of a "floating" link to a useful document.

There are lots of instances of "Quick Links" linking to a PDF document.  I've asked the author if I can move the link into QuickLinks.

**Comment by IanMayo** [comment]
Author agreed to the move. Ian to move link to Quick Links

**Comment by IanMayo** [comment]
Moved.


Mock content fixed in #483
